<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="EmpInfoGroup">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>721</width>
      <height>441</height>
     </rect>
    </property>
    <property name="title">
     <string>EmplInfo</string>
    </property>
    <widget class="QPushButton" name="Insert">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>400</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Insert</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Show">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>400</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Show</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Delete">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>400</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SearchB">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>400</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>400</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>541</width>
       <height>192</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>EmpType</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>SocialSecurityNumber</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>MonthlyCompensation</string>
      </property>
      <property name="textAlignment">
       <set>AlignJustify|AlignVCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>HourlyCompensation</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>DoneHours</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Bonus%</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>RealizedOutcome</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>OutcomeClaim</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="Emp_OK">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>51</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>72</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="NameLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>350</y>
       <width>50</width>
       <height>24</height>
      </rect>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="SCONlabe">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>350</y>
       <width>41</width>
       <height>24</height>
      </rect>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string>Scon</string>
     </property>
     <property name="maxLength">
      <number>32767</number>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="McompLabel">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>350</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>MonthlyComp</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="HdoneLabel">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>350</y>
       <width>71</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>HourDone</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="HcompLabel">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>350</y>
       <width>51</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>HrComp</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="BonusLabel">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>350</y>
       <width>51</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Bonus</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="RealClaimLabel">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>350</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>RealClaim</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="ClaimOClabel">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>350</y>
       <width>81</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Claimoutcome</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="MonthEN">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>350</y>
       <width>41</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Month</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="eMonth">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>310</y>
       <width>41</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="EmpclaimOutcoe">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>310</y>
       <width>51</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="EmprealClaim">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>310</y>
       <width>51</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="Empbonus">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>310</y>
       <width>41</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="EmphrComp">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>310</y>
       <width>41</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="EmphrDone">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>310</y>
       <width>41</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="EmpmComp">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>310</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="Empscon">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>310</y>
       <width>41</width>
       <height>24</height>
      </rect>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>32767</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="Empname">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>310</y>
       <width>81</width>
       <height>24</height>
      </rect>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="LoginBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>-1</y>
      <width>631</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Login</string>
    </property>
    <widget class="QTextEdit" name="userNameTE">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="pwdEdit">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>60</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>UserName</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="LoginButton">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>60</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
